messages/snapshot.create.md

```diff
@@ -1,47 +1,37 @@
 # summary
 
-create a snapshot of a scratch org
+Create a snapshot of a scratch org.
 
 # description
 
-create a snapshot of a scratch org
 A snapshot is a point-in-time copy of a scratch org. The copy is stored in Salesforce and referenced by its unique name in a scratch org definition file.
```

Comment:
I think I know what we meant by "stored in Salesforce" but I don't think it's necessary. So I'm going to suggest an edit.